airflow-core/src/airflow/ui/src/components/DagActions/RunBackfillForm.tsx

```diff
@@ -39,14 +42,16 @@ type RunBackfillFormProps = {
 };
 const today = new Date().toISOString().slice(0, 16);
 
+type BackfillFormProps = DagRunTriggerParams & Omit<BackfillPostBody, "dag_run_conf">;
+
 const RunBackfillForm = ({ dag, onClose }: RunBackfillFormProps) => {
   const [errors, setErrors] = useState<{ conf?: string; date?: unknown }>({});
   const [unpause, setUnpause] = useState(true);
-
-  const { control, handleSubmit, reset, watch } = useForm<BackfillPostBody>({
+  const initialParamsDict = useDagParams(dag.dag_id, true);
+  const { control, handleSubmit, reset, watch } = useForm<BackfillFormProps>({
     defaultValues: {
+      conf: "",
```

Comment:
We need to fetch the conf from the param store in order to keep the Run Parameters form in sync with the json editor.

`const { conf } = useParamStore();`

and

```
  // Automatically reset form when conf is fetched
  useEffect(() => {
    if (conf) {
      reset({ conf });
    }
  }, [conf, reset]);
  ```


```suggestion
      conf,
```

Reply:
Updated.
Thanks for pointing out, I forget to add that.

Reply:
btw I have a small question is that should I resolve the conversation after making the changes, or should I wait for the reviewer to resolve it? Not sure about the convention and I think both have its pros and cons, thus to ask the question. 

Thanks in advance for your help!

Reply:
If you implemented what the reviewer suggested. I think resolving it yourself is ok.